server/shared/storage/index.js

```diff
@@ -54,6 +54,10 @@ class Storage {
     return this.provider.copyFile(key, newKey, options);
   }
 
+  getAssetsPath(repositoryId) {
+    return `repository/${repositoryId}/${config.path}`;
```

Comment:
Use `path.join` instead. It's already imported and it's semantically better. Also be sure that all arguments are strings 😉